exercises/practice/allergies/allergies_test.rb

```diff
@@ -2,303 +2,313 @@
 require_relative 'allergies'
 
 class AllergiesTest < Minitest::Test
-  def test_testing_for_eggs_allergy_not_allergic_to_anything
+  def test_not_allergic_to_anything
     # skip
     allergies = Allergies.new(0)
-    refute allergies.allergic_to?("eggs"), "Tom is not allergic, but it says he is."
+    refute allergies.allergic_to?("eggs"), "Tom is allergic, but it says he is not."
```

Comment:
Instead of changing the wording, change from refute to assert?   A bit of mental gymnastics happening here. ;)